pruning/src/lib.rs

```diff
@@ -126,6 +126,14 @@ impl PruningSeed {
         }
     }
 
+    /// Returns if a peer with this pruning seed should have a non-pruned version of a block.
+    pub fn has_full_block(&self, height: u64, blockchain_height: u64) -> bool {
```

Comment:
Is `blockchain_height` the current height? I assumed so but `CRYPTONOTE_MAX_BLOCK_HEIGHT` is passed elsewhere.
```suggestion
    /// Returns `true` if a peer with this pruning seed should have a non-pruned version of a block.
    pub fn has_full_block(&self, height: u64, blockchain_height: u64) -> bool {
```

Reply:
It's the height of whatever peer we are doing calculations for.

Using a value of `CRYPTONOTE_MAX_BLOCK_HEIGHT` just means we don't take into account the tip blocks which are non-pruned

Reply:
This diff was not applied btw